frontend/src/metabase/search/components/filters/TypeFilter/TypeFilterContent.unit.spec.tsx

```diff
@@ -1,36 +1,36 @@
-import { useState } from "react";
 import userEvent from "@testing-library/user-event";
-import { renderWithProviders, screen, waitFor, within } from "__support__/ui";
+import { useState } from "react";
+import { renderWithProviders, within, screen, waitFor } from "__support__/ui";
+import {
+  createMockDatabase,
+  createMockSearchResult,
+} from "metabase-types/api/mocks";
 import { setupSearchEndpoints } from "__support__/server-mocks";
-import { createMockSearchResult } from "metabase-types/api/mocks";
-import { TypeFilter } from "metabase/search/components/SearchFilterSidebar/filters/TypeFilter";
+import type { EnabledSearchModelType } from "metabase/search/types";
 import type { SearchModelType } from "metabase-types/api";
+import { TypeFilterContent } from "./TypeFilterContent";
 
-const TRANSLATED_NAME_BY_MODEL_TYPE: Record<string, string> = {
+const MODEL_NAME: Record<EnabledSearchModelType, string> = {
+  action: "Action",
   card: "Question",
   collection: "Collection",
   dashboard: "Dashboard",
   database: "Database",
   dataset: "Model",
-  metric: "Metric",
-  pulse: "Pulse",
-  segment: "Segment",
   table: "Table",
 };
 
-const TEST_TYPES: Array<SearchModelType> = [
-  "card",
+const TEST_TYPES: Array<EnabledSearchModelType> = [
   "collection",
   "dashboard",
+  "card",
   "database",
-  "dataset",
   "table",
-  "pulse",
-  "segment",
-  "metric",
+  "dataset",
+  "action",
 ];
 
-const TEST_TYPE_SUBSET: Array<SearchModelType> = [
+const TEST_TYPE_SUBSET: Array<EnabledSearchModelType> = [
```

Comment:
Changed the type to `EnabledSearchModelType` so we get type errors if we try to use `metric` or `segment` without a deliberate change